src/mavedb/view_models/score_set.py

```diff
@@ -260,3 +259,17 @@ class AdminScoreSet(ScoreSet):
 
     normalised: bool
     approved: bool
+
+
+class ScoreSetPublicDump(SavedScoreSet):
+    """Score set view model containing properties to include in a dump of all published data."""
+
+    doi_identifiers: Sequence[DoiIdentifier]
+    primary_publication_identifiers: Sequence[PublicationIdentifier]
+    secondary_publication_identifiers: Sequence[PublicationIdentifier]
+    created_by: Optional[User]
+    modified_by: Optional[User]
+    target_genes: Sequence[TargetGene]
+    private: bool
+    processing_state: Optional[ProcessingState]
+    processing_errors: Optional[dict]
```

Comment:
not sure these should be included, see above comment

Reply:
I didn't like this much either, but my considerations were:
- Consistency with the API seemed like a useful goal to avoid confusing users.
- And there might be some future use case for a complete export in the same format.

I'll defer to your judgment, though.